### DISCUSSION TITLE
Use microphone when streaming from Logic Pro to zoom

### DISCUSSION THREAD

**Discussion by alex-3881** [ExistentialAudio/BlackHole#378] [Q&A]
Hi all! I need to be able to do zoom calls with the sound from logic pro instruments included in the call. 
I did manage to route the logic pro sounds through the blackhole into the zoom call, but I can hear my own voice in the headphones and this makes it incredibly hard to concentrate when playing and singing. Basically, I need to hear what I'm playing in the headphones, but not my own voice. Is that possible? Can't find a solution...

**Comment by devinroth** [comment]
If you use an aggregate you can route the outputs to the appropriate locations.

I don’t have a tutorial for that sorry.

Devin Roth

**Reply by alex-3881** [reply]
Could you please tell me what should go where? I have tried everything it seems, the best I could achieve with the aggregate device is that the person I'm talking to in Zoom can either hear me or the logic pro sounds. I don't want to hear myself when speaking so adding microphone input to logic pro doesn't seem to be a solution - I will hear myself. 
Logic -> blackhole, add blackhole and microphone in aggregate and use aggregate as input in Zoom doesn't work at all - the person can hear neither me nor the Logic sounds... 
![photo_2021-04-21_23-56-15](https://user-images.githubusercontent.com/78366335/115619942-3ab07300-a2fd-11eb-9244-35e71050e758.jpg)
